susemanager-utils/susemanager-sls/salt/switch_to_bundle/mgr_switch_to_venv_minion.sls

```diff
@@ -0,0 +1,19 @@
+{%- set avoid_venv_salt_minion = salt['pillar.get']('mgr_avoid_venv_salt_minion') %}
+{%- set is_salt_ssh = salt['pillar.get']('contact_method') in ['ssh-push', 'ssh-push-tunnel'] %}
+{%- set is_saltboot = salt['file.file_exists']('/etc/ImageVersion') %}
+
+{%- if grains['saltversion'] == '3000' and not is_salt_ssh and 
+       not avoid_venv_salt_minion == true and not is_saltboot %}
```

Comment:
Is there any particular reason to use `true` here instead of `True`? 

I think both works fine as jinja2 evaluates both as `True`, but I would prefer a more Pythonic `True` here.

Reply:
Sorry, I missed your comment mentioning that you explicitely use `true` Jinja2 literal. I'm fine with it.